build.gradle

```
@@ -229,9 +229,7 @@ subprojects { subproject ->
         // To avoid compiler warnings about @API annotations in JUnit5 code.
         testCompileOnly 'org.apiguardian:apiguardian-api:1.0.0'
         testCompile "org.postgresql:postgresql:42.2.16"
-        testCompile (group: 'com.opentable.components', name: 'otj-pg-embedded', version: '0.13.3') {
-            exclude group: 'org.postgresql', module: 'postgresql'
-        }
+        testCompile "org.testcontainers:postgresql:1.15.0"
```

See https://github.com/hmcts/ccd-definition-store-api/pull/899
```suggestion
        testCompile "org.testcontainers:postgresql:1.15.1"
```